source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/RendererUtil.java

```diff
@@ -33,6 +33,12 @@ public static Calendar getDate(String s)
         return calendar;
     }
 
+    public static boolean isValidTime(String s)
```

Comment:
Question: How come no equivalent check for Date values? Do we validate that earlier on or elsewhere ? Also what date formats are we allowing again - per our schema looks like - "YYYY-MM-DD" ?

Reply:
added the mirrored change